reaktive/src/commonTest/kotlin/com/badoo/reaktive/observable/TakeWhilePredicateTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.test.base.assertError
+import com.badoo.reaktive.test.observable.TestObservable
+import com.badoo.reaktive.test.observable.assertComplete
+import com.badoo.reaktive.test.observable.assertValues
+import com.badoo.reaktive.test.observable.onNext
+import com.badoo.reaktive.test.observable.test
+import kotlin.test.Test
+
+class TakeWhilePredicateTest : ObservableToObservableTests by ObservableToObservableTestsImpl({ takeWhile { true } }) {
```

Comment:
I'm slightly surprised I only had to write 3 tests in this class. Is everything covered by generic base tests?

Reply:
That's ok, generic tests do the rest.